sdks/python/setup.py

```diff
@@ -259,11 +259,17 @@ def run(self):
 
 python_requires = '>=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*,!=3.4.*'
 
-if sys.version_info[0] == 2:
+if sys.version_info.major == 2:
   warnings.warn(
       'You are using Apache Beam with Python 2. '
       'New releases of Apache Beam will soon support Python 3 only.')
 
+if sys.version_info.major == 3 and sys.version_info.minor >= 8:
```

Comment:
you're including 3.8 in this range. Is that intended?

Reply:
@pabloem Yes, it's intended, because our Py3.8 test coverage is not yet up to the same standard as other versions, but we are working on that (adding Postcommit tests in #11788 and better test coverage strategy: BEAM-9943)